src/main/java/DukeExceptions.java

```diff
@@ -0,0 +1,3 @@
+public class DukeExceptions {
```

Comment:
You can consider making use of this class.
```suggestion
public class DukeException extends Exception {
```